docs/reference/authorization.md

```diff
@@ -1,26 +1,23 @@
 ---
 layout: default
-title: Authorization
-description: This section covers authorization (using AWS IAM)  of your lakeFS server.
+title: Role Based Access Control (RBAC)
+description: This section covers authorization (using RBAC)  of your lakeFS server.
 parent: Reference
 nav_order: 65
 has_children: false
 ---
 
 
-# Authorization
+# Role Based Access Control (RBAC)
 {: .no_toc }
 
 {% include toc.html %}
 
-## Role Based Access Control
+{: .note}
+> RBAC is only available in [lakeFS cloud](https://lakefs.cloud). This scalable fully-managed lakeFS service is also SOC 2 compliant.<br/>
+> A simpler ACL-based authorization mechanism is planned for open-source lakeFS. For more details see [this announcement](../posts/security_update.html). 
```

Comment:
Thinking about it some more, I think we should consider:
```suggestion
> A simpler ACL-based authorization mechanism is planned for open-source lakeFS. For more details see [this announcement](/posts/security_update.html). 
```

Per [RFC 1808](https://datatracker.ietf.org/doc/html/rfc1808#section-5.1), this should resolve from https://host.domain/path/to/docs/reference/authorization.md to https://host.domain/posts/security_update.html.  So it will continue to work if we change the depth of docs/reference/.